src/main/java/frc/robot/commands/rightIntakeGoTo.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants;
+
+public class rightIntakeGoTo extends CommandBase {
+  /** Creates a new IntakeGoTo. */
+  public double endpoint;
+ //tracking initial start point so we can compare during climbing to see when to finish
+  public double leftStartpoint;
+  public double rightStartpoint;
+  public rightIntakeGoTo(double Endpoint) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    endpoint = -Endpoint;
+
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    RobotContainer.m_intakeSubsystem.stopActuators();
```

Comment:
TEST this to  make surte the right intaker arm isnt stopping the left intake arm and vice versa